lib/tasks/journeys_report.rake

```diff
@@ -50,16 +50,19 @@ namespace :journeys do
           }
         end
         person = move.profile.present? ? move.profile.person : nil
+        move_notification = move.notifications.webhooks.where(event_type: 'create_move').order(:delivered_at).first
         moves << {
           id: move.id,
           supplier: supplier.key,
           reference: move.reference,
-          notification_date: move.created_at, # TODO: use notifications table
+          notified_at: move_notification&.delivered_at,
           updated_at: move.updated_at,
           move_date: move.date,
           from: move.from_location.nomis_agency_id,
           to: move.to_location&.nomis_agency_id,
           person_id: person&.id,
+          pnc_number: person&.police_national_computer,
+          prison_id: person&.nomis_prison_number,
```

Comment:
We also have both `prison.number` and `nomis_prison_number`, this one seems to more explicit map the requirement